### ISSUE TITLE
Questions about datagen.py

### ISSUE THREAD

**Issue by yzldw333** [kuangliu/pytorch-retinanet#22]
Hi, I met a question that When I want to use this project to train WIDER-Face-Detection dataset. And I have create label to meet your script like this(name.jpg xmin ymin xmax ymax), When I want to debug to see whether I am right, I saw it print the data shape from dataloader like this.
(1L, 3L, 600L, 600L)
(1L, 67995L, 4L)
(1L, 67995L)
why it becomes 67995 dimension, I found that DataSet __getitem__ method create data shape is
(1L, 3L, 600L, 600L)
(1L, 4L)
(1L)
I did'n use pytorch before, is something wrong with pytorch? or my label file?

**Comment by yzldw333** [comment]
Sorry to bother you, I read the paper carefully and got it.
